src/sail_lean_backend/pretty_print_lean.ml

```diff
@@ -65,7 +65,34 @@ let rec untuple_args_pat typs (P_aux (paux, ((l, _) as annot)) as pat) =
   | _, _ -> unreachable l __POS__ "Unexpected pattern/type combination"
 
 let doc_typ (Typ_aux (t, _) as typ) =
-  match t with Typ_id (Id_aux (Id "unit", _)) -> string "Unit" | _ -> failwith "Type not translatable yet."
+  match t with
+  | Typ_id (Id_aux (Id "unit", _)) -> string "Unit"
+  | Typ_id (Id_aux (Id "int", _)) -> string "Int"
```

Comment:
Note that there are several integer types, `int`, `atom`, `range`, and `implicit`.  The last two only appear in certain situations (actual ranges, and implicit arguments, respectively), but `atom` is the original name for "the integer that matches this type variable" and can appear unexpectedly.